src/mds/Migrator.cc

```diff
@@ -3591,6 +3595,8 @@ void Migrator::handle_export_caps(const cref_t<MExportCaps> &ex)
     return;
   }
 
+  if (mds->logger) mds->logger->inc(l_mdss_handle_export_caps);
+
```

Comment:
remove all change in this file. they are not cap message

Reply:
Done